src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENAUITests.xcscheme

```diff
@@ -44,12 +44,20 @@
       buildConfiguration = "UITesting"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
       launchStyle = "0"
       useCustomWorkingDirectory = "NO"
       ignoresPersistentStateOnLaunch = "NO"
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
+      <EnvironmentVariables>
```

Comment:
Do we need this environment variable?

Reply:
Looking at the CircleCI logs,

    xcodebuild -workspace ./ENA.xcworkspace -scheme ENAUITests

I don't think we do. Will delete, commit & push.